src/Item.php

```diff
@@ -13,10 +13,26 @@ public function getTaxRate()
     }
 
     /**
+     * @inheritDoc
+     */
+    public function getTotalTaxAmount()
```

Comment:
This getters/setter combo is never used. Item transformation uses a calculated value `$item->getQuantity() * $item->getPrice() * 100`